package.json

```diff
@@ -17,6 +17,7 @@
     "lint": "prettier --check '{lib,test}/**/*.ts' && tslint --format stylish '{lib,test}/**/*.ts'",
     "format": "prettier --loglevel warn --write '{lib,test}/**/*.ts' && tslint --fix --format stylish '{lib,test}/**/*.ts'",
     "test": "npm run lint && npm run unit-test",
+    "test-windows": "prettier --check \"{lib,test}/**/*.ts\" && tap test/windows -R=spec --timeout=300",
```

Comment:
should the `prettier` part be in here?

Reply:
well, actually I can just `npm run lint` instead 👍